tests/test_usage.py

```diff
@@ -118,6 +134,40 @@ def test_multicast_help_arg_main(self):
 			theResult = False
 		self.assertTrue(theResult, fail_fixture)
 
+	def test_multicast_message_send_recv(self):
```

Comment:
🙅🏻 rename as per project convention
```suggestion
	def test_hear_works_WHEN_say_works(self):
```

Reply:
see #34